templates/dashboard/index.html

```diff
@@ -398,7 +398,7 @@
       {% endif %}
       <span class="alias-activity">{{ alias_info.nb_forward }}</span> forwards,
```

Comment:
should we change `forwards -> forwarded, blocks -> blocked` as well?

Reply:
Yes, good catch! Resolved in latest commit.